Resfeber/View Controllers/AddTripViewController.swift

```diff
@@ -0,0 +1,159 @@
+//
+//  AddTripViewController.swift
+//  Resfeber
+//
+//  Created by Joshua Rutkowski on 12/5/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import CoreData
+
+class AddTripViewController: UIViewController {
+    
+    // MARK: - Properties
+    var tripService: TripService!
+    var coreDataStack = CoreDataStack()
```

Comment:
You should create the CoreDataStack as a shared instance accessed as a Singleton

Reply:
Thank you for the advice! Changes have been made.